client.py

```diff
@@ -0,0 +1,107 @@
+import logging
+import time
+from datetime import datetime
+
+import click
+import requests
+from apex_ocr.engine import ApexOCREngine, SummaryType
+from apex_ocr.roi import scale_rois
+from apex_ocr.utils import get_primary_monitor
+from config import DATA_DIRECTORY, LOG_DIRECTORY
+from PIL import Image, ImageGrab
+from rich.logging import RichHandler
+
+logging.captureWarnings(True)
+logger = logging.getLogger(__name__)
+
+# TODO: Remove this and get this from Apex ROI
+PRIMARY_MONITOR = get_primary_monitor()
+PRIMARY_BBOX = (
+    PRIMARY_MONITOR.x,
+    PRIMARY_MONITOR.y,
+    PRIMARY_MONITOR.x + PRIMARY_MONITOR.width,
+    PRIMARY_MONITOR.y + PRIMARY_MONITOR.height,
+)
+
```

Comment:
Can you add a signal handler for ctrl-c?

Reply:
Should gracefully handle keyboard interrupts now. Had to replace click with `argparse` because `click` doesn't seem to allow overriding its custom exception handler, which exits and prints "Aborted!" by default.